Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -33,10 +33,10 @@ public typealias PurchaseResultData = (transaction: StoreTransaction?,
 /**
  Completion block for ``Purchases/purchase(product:completion:)``
  */
-public typealias PurchaseCompletedBlock = @MainActor @Sendable (StoreTransaction?,
-                                                                CustomerInfo?,
-                                                                PublicError?,
-                                                                Bool) -> Void
+public typealias PurchaseCompletedBlock =  @Sendable (StoreTransaction?,
+                                                      CustomerInfo?,
+                                                      PublicError?,
+                                                      Bool) -> Void
```

Comment:
I needed to remove this reference otherwise we'd still get a crash, since we have places where we store collections of these `@MainActor` references.